roles/nvidia-dgx-diag/tasks/get-firmware.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Get on-board and container firmware versions
+  shell: "docker run --privileged --rm -v /:/hostfs {{ firmware_update_repo }}:{{ firmware_update_tag }} show_version | tee {{ nv_diag_dir }}/fw-versions.log"
```

Comment:
Make sure this works on several Ansible configurations, document where it does not